components/edit-collective/Menu.js

```diff
@@ -267,7 +267,7 @@ const EditCollectiveMenu = ({ collective, selectedSection }) => {
             renderMenuItem(getSectionInfo(EDIT_COLLECTIVE_SECTIONS.HOST_PLAN))}
           {[USER, ORGANIZATION].includes(collective.type) &&
             renderMenuItem(getSectionInfo(EDIT_COLLECTIVE_SECTIONS.HOST_METRICS))}
-          {[USER, ORGANIZATION].includes(collective.type) &&
+          {[USER].includes(collective.type) &&
```

Comment:
Maybe just try to display `POLICIES` here for `ORGANIZATION` and `USER`.

Reply:
@znarf thank you for the review, on it. Should i then remove the code related to `EXPENSE_PAYOUTS` cause they aren't used anywhere except here.

Reply:
If it's not used, sure, let's delete it!